src/index.html

```diff
@@ -1,14 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements.
```suggestion
<html lang="en">
  <head>
```